tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -299,6 +278,35 @@ <h3>No precision–recall curve data was found.</h3>
         const defaultDisplayName = tag.replace(/\/pr_curves$/, '');
         return tf_utils.aggregateTagInfo(runToTagInfo, defaultDisplayName);
       },
+      _createDataChangeCallback(tag) {
+        return function onDataChange(runToData) {
+          this.set(
+            '_tagToRunToData',
+            Object.assign({}, this._tagToRunToData, {[tag]: runToData})
+          );
+        }.bind(this);
+      },
+      _computeRunToAvailableTimeEntries(tagToRunToData) {
+        const canonicalTag = {}; // map from run to canonical tag name
+        for (const [tag, runToData] of Object.entries(tagToRunToData)) {
+          for (const [run, data] of Object.entries(runToData)) {
+            // arbitrary stable ordering: smallest tag name is canonical
+            if (canonicalTag[run] == null || tag < canonicalTag[run]) {
```

Comment:
No AI: `tag < canonicalTag[run]` may want to use the [compareTagNames](https://github.com/tensorflow/tensorboard/blob/b929faea0eb43b1fc61e8b30398423191a40ec73/tensorboard/components/vz_sorting/sorting.ts#L26) ideally but I assume our Python backend did not do that either.

Reply:
Correct that the Python backend didn’t, and I don’t think it’s super
important. The intent is that it shouldn’t matter which tag is picked,
so I have a weak preference to keeping this simple.